Document/0x05b-Basic-Security_Testing.md

```diff
@@ -105,9 +105,30 @@ An extensive list of root detection methods is presented in the "Testing Anti-Re
 
 For a typical mobile app security build, you'll usually want to test a debug build with root detection disabled. If such a build is not available for testing, you can disable root detection in a variety of ways that will be introduced later in this book.
 
-#### Recommended Tools
-
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1227>
+#### Recommended Tools - Android device
+
+There are many tools and frameworks used throughout this guide to assess the security of Android applications. In the next sections you will learn more about some of their main commands and interesting use cases. Please check the official documentation for installation instructions of the following tools/APKs:
+
+- APK Extractor: App to extract APKs without root.
+- Frida server: Server for Frida, the dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers.
+- Drozer agent: Agent for drozer, the framework that allows you to search for security vulnerabilities in apps and devices.
+- Xposed: Framework that can change the behavior of the system and apps without touching any APKs.
+
+#### Recommended Tools - Host computer
+
+In order to analyze Android apps, you should install the following tools on your host computer. Please check the official documentation for installation instructions of the following tools/frameworks. We'll be referring to them throughout the guide.
+
+- Adb: Used for communication with Android devices and debugging applications.
```

Comment:
I would not have a list here and below start with the tools again. It would be better to _mimic_ chapter 0x6b and start right away with the subsections after the previous little intro.

Reply:
Do you mean subsections describing the tools like in the iOS part https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06b-Basic-Security-Testing.md#recommended-tools---host-computer
If so, yes agree

Reply:
Yes, exactly :)

Reply:
Ok, got it. Let me update this quickly.